src/uploader.js

```diff
@@ -80,6 +84,7 @@ const useWidget = (
       if (locale) delete window.UPLOADCARE_LOCALE
       if (localePluralize) delete window.UPLOADCARE_LOCALE_PLURALIZE
       if (localeTranslations) delete window.UPLOADCARE_LOCALE_TRANSLATIONS
+      if (previewUrlCallback) delete window.UPLOADCARE_PREVIEW_URL_CALLBACK
     }
   }, [locale, localeTranslations, localePluralize])
```

Comment:
use hook useCallback for set function in deps.
previewUrlCallback

Reply:
😨 fixed